variables.tf

```diff
@@ -50,3 +39,46 @@ variable "automation_account" {
   description = "Automation account where the resource graph script will be deployed."
   nullable    = false
 }
+
+variable "event_pipeline_config" {
+  type = object({
+    enabled     = bool
+    name        = optional(string)
+    service_uri = optional(string)
+  })
+  description = <<-DOC
+  ```
+  "<group_name>" = {
+    azuread_role_assignable = optional(string)    (if you want to assign Azure AD roles to the group) 
+    role_assignments = {
+      "<role_assignment>" = [                 (must be a role_definition_name or role_definition_id from azure)
+        "<scope>"                             (every element must be a scope: "mg:<mg_id>", "sub:<subscription_id>", "root" for Tenant Root Group or a full scope ID)
+      ]
+    }
+}
+  ```
+  DOC
+
+  description = <<-DOC
+  ```
+  {
+    enabled       = Enable the action group if you want to send data to a monitoring service.
+    name          = Name of the alert webhook.
+    service_uri   = Link to the webhook receiver URL. Must contain the placeholder \"{{pipeline_key}}\" which will be replaced with the secret from var.event_pipeline_key.
+}
+  ```
+  DOC
+  default = {
+    enabled = false
+  }
```

Comment:
Defining description two times lookswrong

Reply:
Copy paste mistake. We noticed that too :)